packages/components/src/globals/scss/_css--reset.scss

```diff
@@ -193,6 +193,11 @@
 
     * {
       box-sizing: border-box;
+
+      &:after,
+      &:before {
+        box-sizing: border-box;
```

Comment:
Any way without the change here? Sounds that components-wide re-testing may be needed if we go with this change.

Reply:
@asudoh thanks for your note! I'll revisit this today 👀 

Quick question -- is the use of `box-sizing: border-box` as a standalone rule in specific situations (like this one) acceptable at all? (Only asking because it seems like the path of least resistance & won't require any tests outside of this component 😅 )

Reply:
Good question @jendowns - Assuming your questions about adding `box-sizing: border-box` to checkbox style, I think it's OK as long as there is a comment describing what condition it's required (specific `$css--reset` variable IIUC) and why.